2021-09-06 22:36:47,618 INFO    MainThread:19832 [wandb_setup.py:_flush():69] setting env: {'entity': 'zumo09', 'project': 'flatland-rl-global-reward-sweep', 'sweep_id': 'z806yjo8', 'root_dir': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN', 'run_id': 'm4l9a4dd', 'sweep_param_path': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN\\wandb\\sweep-z806yjo8\\config-m4l9a4dd.yaml'}
2021-09-06 22:36:47,618 INFO    MainThread:19832 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-06 22:36:47,618 INFO    MainThread:19832 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-06 22:36:47,618 INFO    MainThread:19832 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-06 22:36:47,619 INFO    MainThread:19832 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210906_223647-m4l9a4dd\logs\debug.log
2021-09-06 22:36:47,619 INFO    MainThread:19832 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210906_223647-m4l9a4dd\logs\debug-internal.log
2021-09-06 22:36:47,619 INFO    MainThread:19832 [wandb_init.py:init():375] calling init triggers
2021-09-06 22:36:47,619 INFO    MainThread:19832 [wandb_init.py:init():382] wandb.init called with sweep_config: {'global_reward': 5, 'invalid_action_penalty': -1, 'step_penalty': -1.5}
config: {'n_episodes': 500, 'env_config': 0, 'n_evaluation_episodes': 5, 'checkpoint_interval': 100, 'eps_start': 1.0, 'eps_end': 0.01, 'eps_decay': 0.99, 'invalid_action_penalty': -1.0, 'step_penalty': -1.5, 'global_reward': 5.0, 'stop_penalty': 0.0, 'start_penalty': 0.0, 'buffer_size': 100000, 'buffer_min_size': 0, 'restore_replay_buffer': '', 'save_replay_buffer': False, 'batch_size': 128, 'gamma': 0.93, 'tau': 0.001, 'learning_rate': 8e-05, 'hidden_size': 128, 'update_every': 8, 'render': False, 'training_id': '210906223645', 'n_agents': 2, 'x_dim': 35, 'y_dim': 35, 'n_cities': 2, 'max_rails_between_cities': 2, 'max_rails_in_city': 3, 'malfunction_rate': 0.02, 'seed': 0, 'observation_tree_depth': 2, 'observation_radius': 10, 'observation_max_path_depth': 30}
2021-09-06 22:36:47,619 INFO    MainThread:19832 [wandb_init.py:init():424] starting backend
2021-09-06 22:36:47,619 INFO    MainThread:19832 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-09-06 22:36:47,626 INFO    MainThread:19832 [backend.py:ensure_launched():135] starting backend process...
2021-09-06 22:36:47,710 INFO    MainThread:19832 [backend.py:ensure_launched():140] started backend process with pid: 19736
2021-09-06 22:36:47,711 INFO    MainThread:19832 [wandb_init.py:init():429] backend started and connected
2021-09-06 22:36:47,712 INFO    MainThread:19832 [wandb_run.py:_config_callback():843] config_cb None None {'global_reward': 5, 'invalid_action_penalty': -1, 'step_penalty': -1.5}
2021-09-06 22:36:47,714 INFO    MainThread:19832 [wandb_init.py:init():477] updated telemetry
2021-09-06 22:36:47,714 INFO    MainThread:19832 [wandb_init.py:init():500] communicating current version
2021-09-06 22:36:49,461 INFO    MainThread:19832 [wandb_init.py:init():505] got version response 
2021-09-06 22:36:49,461 INFO    MainThread:19832 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-06 22:36:49,855 INFO    MainThread:19832 [wandb_init.py:init():540] starting run threads in backend
2021-09-06 22:36:52,237 INFO    MainThread:19832 [wandb_run.py:_console_start():1601] atexit reg
2021-09-06 22:36:52,239 INFO    MainThread:19832 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-06 22:36:52,240 INFO    MainThread:19832 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-06 22:36:52,242 INFO    MainThread:19832 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-06 22:36:52,242 INFO    MainThread:19832 [wandb_init.py:init():565] run started, returning control to user process
2021-09-06 23:37:19,906 INFO    MainThread:19832 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-06 23:37:19,907 INFO    MainThread:19832 [wandb_run.py:_restore():1543] restore
2021-09-06 23:37:20,491 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2021-09-06 23:37:20,809 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2021-09-06 23:37:21,247 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 9108
}

2021-09-06 23:37:21,358 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 9537
}

2021-09-06 23:37:21,470 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 9537
}

2021-09-06 23:37:21,582 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:21,692 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:21,800 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:21,908 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:22,016 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:22,126 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:22,570 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:22,684 INFO    MainThread:19832 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9537
  total_bytes: 9537
}

2021-09-06 23:37:23,982 INFO    MainThread:19832 [wandb_run.py:_show_summary():1848] rendering summary
2021-09-06 23:37:23,985 INFO    MainThread:19832 [wandb_run.py:_show_history():1886] rendering history
2021-09-06 23:37:23,991 INFO    MainThread:19832 [wandb_run.py:_show_files():1915] logging synced files
